### PR TITLE
Kuntal: tasks89-92_SearchQA_and_ClimateFever

### PR DESCRIPTION
**Pull request by Rubicon1887**

### CODE REVIEW

**Comment by swarooprm** [comment]
@kuntalkumarpal, wondering if you can help review this PR?

File: tasks/task89_SearchQA_category_classification.json [hunk 1]
```diff
@@ -0,0 +1,1238 @@
+{
+    "Contributors": [
+        "Atharva Anand Barwe"
+    ],
+    "Source": [
+        "SearchQA (https://arxiv.org/abs/1704.05179)"
+    ],
+    "Categories": [
+        "Category Classfication"
```
Comment (kuntalkumarpal) [inline tasks/task89_SearchQA_category_classification.json:9]: Just write "Classification"

File: tasks/task89_SearchQA_category_classification.json [hunk 2]
```diff
@@ -0,0 +1,1238 @@
+{
+    "Contributors": [
+        "Atharva Anand Barwe"
+    ],
+    "Source": [
+        "SearchQA (https://arxiv.org/abs/1704.05179)"
```
Comment (kuntalkumarpal) [inline tasks/task89_SearchQA_category_classification.json:6]: also add data link

**Comment by kuntalkumarpal** [comment]
@swarooprm Looks good.
Just let the student know whether to keep the squad entries in the Readme file or not

**Comment by swarooprm** [comment]
@Rubicon1887, keep squad entries in readme. 
Can you address reviewer comments so that I can evaluate?

**Comment by Rubicon1887** [comment]
Understood. I'll make the corrections.

**Comment by Rubicon1887** [comment]
I have made changes according to the reviewer's comments. I added
"Category" and the correct dataset links and reuploaded the 4 json files.
Also, here are screenshots of the readme files from main and my fork
(athNLP). Are these changes correct?

**Comment by swarooprm** [comment]
@kuntalkumarpal, check if you are happy or have any other feedback?

**Comment by kuntalkumarpal** [comment]
Looks good to me

**Comment by swarooprm** [comment]
Your HW2 is complete now.

Would you like to submit the same tasks to our big project repo https://github.com/allenai/natural-instructions-expansion/pulls? You just need to fork that repo, rename tasks starting from the latest number in the repo and create PR there. This way, you can be part of our research on "learning from instructions instead of learning from examples (conventional machine learning)". This is purely optional and your HW2 grade does not depend on this @Rubicon1887

**Comment by swarooprm** [comment]
@Rubicon1887 Just a reminder that, if you plan to submit these tasks to the main Github repo and join our research project, Monday (27th Sept) is the deadline. Again, this is optional and independent of your HW2 grade.

**Comment by Rubicon1887** [comment]
I forked the repo, uploaded my tasks, updated those tasks in the readme and
then created the pull request. It says that a review is required by
reviewers. Also, it shows that a check has failed - "uppercase alphabet
found." this wasn't an issue earlier during submission.

**Pull request closed without merging**